### PR TITLE
Add Amazon EKS to helm docs topics kubernetes_distros

### PR DESCRIPTION
**Pull request by bcdady**
Add Amazon AWS Elastic Kubernetes Service (EKS) to the list, with link to Amazon EKS documentation titled "Using Helm with Amazon EKS".
Signed-off-by: Bryan Dady <bryan@dady.us>

### CODE REVIEW

**Comment by bcdady** [comment]
[I tried to sign-off properly](https://github.com/bcdady/helm-www/commit/e4879257695c22bdfc4f4248b16aeedd232b6fd3), but apparently messed it up somehow.
I then followed the steps `git commit --amend --signoff` and `git push --force-with-lease origin patch-2`, but it appears that's still not satisfying the checks for this PR. Any advice on how to more properly comply with the contribution guidelines are welcomed.

**Comment by jdolitsky** [comment]
@bcdady Please see this error: https://github.com/helm/helm-www/pull/748/checks?check_run_id=859104683

Appears related to the email address you used in signoff mismatching

**Comment by bacongobbler** [comment]
@bcdady in order for the DCO sign-off check to work, I believe you have to link your email address to your github account. Once that's done we can re-run the DCO sign-off check.

More info here: https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/adding-an-email-address-to-your-github-account

**Comment by bcdady** [comment]
I had (apparently at some point) turned on email privacy in my GitHub settings, and so that's why GitHub was using that github.com email address in my previous commit message. I change my account settings and then re-force-pushed, but then re-read what you wrote, that you'd need to "re-run the DCO sign-off check". I expect that should pass now.

**Comment by bcdady** [comment]
Apparently changing my email privacy (in GitHub settings) didn't change the expected email address after the initial commit. Thanks for your patience and your help as I've clearly never done a DCO sign like this before.

File: content/en/docs/topics/kubernetes_distros.md [hunk 1]
```diff
@@ -16,14 +16,17 @@ alphabetically) if desired.
 
 ## AKS
 
-Helm works with [Azure Kubernetes
-Service](https://docs.microsoft.com/en-us/azure/aks/kubernetes-helm).
+Helm works with [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/kubernetes-helm).
```
Comment (bacongobbler) [inline content/en/docs/topics/kubernetes_distros.md:19]: in #743 we started to word wrap at 80 characters for easier readability. The prior change was intentional here. Can you please revert this?

File: content/en/docs/topics/kubernetes_distros.md [hunk 2]
```diff
@@ -16,14 +16,17 @@ alphabetically) if desired.
...
 
 ## DC/OS
 
 Helm has been tested and is working on Mesospheres DC/OS 1.11 Kubernetes
 platform, and requires no additional configuration.
 
+## EKS
+
+Helm works with Amazon Elastic Kubernetes Service (Amazon EKS): [Using Helm with Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/helm.html).
```
Comment (bacongobbler) [inline content/en/docs/topics/kubernetes_distros.md:28]: can you please wrap this sentence at 80 characters?

File: content/en/docs/topics/kubernetes_distros.md (hunk 1 above)
Comment (bcdady) [reply content/en/docs/topics/kubernetes_distros.md]: Done.

File: content/en/docs/topics/kubernetes_distros.md (hunk 2 above)
Comment (bcdady) [reply content/en/docs/topics/kubernetes_distros.md]: Done.

**Pull request merged**